test/4_fund_actions.js

```diff
@@ -168,16 +169,16 @@ contract('Fund Actions', (accounts) => {
       const amt = INVESTOR_ALLOCATION;
 
       it('should make subscription request given valid amount', () => {
-        return fund.requestSubscription({ from: investor, value: ethToWei(amt), gas: GAS_AMT })
+        fund.requestSubscription({ from: investor, value: ethToWei(amt), gas: GAS_AMT })
```

Comment:
Just make sure to check all.  This will fail without the return.  So need to change to `() => promise`